packages/components/pie-modal/src/index.ts

```diff
@@ -186,19 +206,20 @@ export class PieModal extends RtlMixin(LitElement) {
     };
 
     /**
-     * Dispatch `ON_MODAL_OPEN_EVENT` event.
-     * To be used whenever we open the modal.
+     * Template for the close button element. Called within the
+     * main render function.
      *
-     * @event
+     * @protected
      */
-    private _dispatchModalOpenEvent = () : void => {
-        const event = new CustomEvent(ON_MODAL_OPEN_EVENT, {
-            bubbles: true,
-            composed: true,
-        });
-
-        this.dispatchEvent(event);
-    };
+    protected renderCloseButton (): TemplateResult {
```

Comment:
Out of interest why is this protected and not private?

Reply:
fixed.

Reply:
I guess the advantage here (although not tested it) is we have access via tests but since it's not being used I have removed.